code/modules/clothing/suits/marine_armor.dm

```diff
@@ -639,9 +639,9 @@
 
 //===========================//DISTRESS\\================================
 
-/obj/item/clothing/suit/storage/marine/veteran/bear
-	name = "\improper H1 Iron Bears vest"
-	desc = "A protective vest worn by Iron Bears mercenaries."
+/obj/item/clothing/suit/storage/marine/veteran/wolves
+	name = "\improper H1 Steel Wolves vest"
+	desc = "A protective vest worn by Steel Wolves mercenaries."
 	icon_state = "bear_armor"
```

Comment:
If you really wanted you could also fix the icon states, though not required

Reply:
I'll fix it tonight, its a fair ask, and id ask someone else to do it too.